s3watcher/package.json

```diff
@@ -9,9 +9,12 @@
     "rx": "^3.1.2",
     "rx-node": "^1.0.1"
   },
-  "devDependencies": {},
+  "devDependencies": {
+      "jscs": "^1.13.0"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+      "test": "npm run lint",
+      "lint": "jscs S3Watcher.js lib"
```

Comment:
https://github.com/guardian/grid/pull/1514 moves the code to a `lambda` directory, so this would soon become `jscs lambda`.